app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("Need to vaccinate.")
+        else:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("Need to vaccinate again.")
+            elif (visitor.get("wearing_a_mask") is None
+                  or visitor.get("wearing_a_mask") is False):
+                raise NotWearingMaskError("Need to wear mask")
+            else:

```

Comment:
The use of `else` after `raise` statements is unnecessary. According to the checklist, you should avoid using unnecessary `else` blocks. You can directly return the welcome message after the checks without using `else`.